src/rand.rs

```diff
@@ -227,6 +252,15 @@ extern {
     fn GFp_sysrand_chunk(buf: *mut u8, len: c::size_t) -> c::long;
 }
 
+#[cfg(any(target_os = "macos", target_os = "ios"))]
+#[link(name = "Security", kind = "framework")]
+extern {
```

Comment:
@briansmith: I tried putting them in the sub-module, but doing so required another `use super::super::c` in the sub-module, which seemed to go against the prevailing style in this project. Would you prefer the sub-mobile-with-a-`use` approach?